src/open_inwoner/scss/views/App.scss

```diff
@@ -254,63 +254,68 @@
 }
 
 @font-face {
-  font-family: 'TheMix C5';
-  src: local('TheMix C5'),
-    url('/static/fonts/TheMixC5/DesktopFonts/TheMixC5-5_Plain.otf')
-      format('opentype');
+  font-family: 'Heading';
+  font-weight: bold;
+  src: url('/static/fonts/ubuntu/Ubuntu-M.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'TheMix C5';
+  font-family: 'Heading';
   font-weight: bold;
-  src: local('TheMix C5'),
-    url('/static/fonts/TheMixC5/DesktopFonts/TheMixC5-7_Bold.otf')
-      format('opentype');
+  font-style: italic;
+  src: url('/static/fonts/ubuntu/Ubuntu-MI.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'TheSans C5';
-  src: local('TheSans C5'),
-    url('/static/fonts/TheSansC5/DesktopFonts/TheSansC5-5_Plain.otf')
-      format('opentype');
+  font-family: 'Body';
+  src: url('/static/fonts/lato/Lato-Regular.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'TheSans C5';
+  font-family: 'Body';
   font-weight: bold;
-  src: local('TheSans C5'),
-    url('/static/fonts/TheSansC5/DesktopFonts/TheSansC5-7_Bold.otf')
-      format('opentype');
+  src: url('/static/fonts/lato/Lato-Bold.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Body';
+  font-style: italic;
+  src: url('/static/fonts/lato/Lato-Italic.ttf') format('truetype');
```

Comment:
@pi-sigma Around here I am specifying separate font-files for the regular/bol/Italic/bold-italic situations, since that is the official way to prevent so-called "faux-rendering" but... I am not sure what effect this will have on your PR #963